CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 ### Fixed
 
 - Seccomp filters are now applied to all Firecracker threads.
+- Enforce minimum length of 1 character for the jailer ID.
+- Exit with error code when starting the jailer process fails.
```

Comment:
nit: 
```suggestion
 - Exit with panic when starting the jailer process fails.
```

Reply:
I think panic is something too rust specific, that's why I wrote error code. What I was trying to say here is that firecracker exit code != 0 in case of error.